warehouse/ingest-core/src/main/java/datawave/ingest/mapreduce/job/reindex/ShardReindexJob.java

```diff
@@ -139,31 +171,72 @@ private Job setupJob() throws IOException, ParseException, AccumuloException, Ta
+        for (String table : configuration.get("job.output.table.names").split(",")) {
+            try {
+                Map<String,String> tableProperties = client.tableOperations().getTableProperties(table);
+                if (tableProperties == null) {
+                    throw new IllegalArgumentException("configured output table: " + table + " does not exist");
+                }
+            } catch (TableNotFoundException tnfe) {
+                throw new IllegalArgumentException("configured output table: " + table + " does not exist");
+            }
+        }
+
         ShardedTableMapFile.setupFile(configuration);
 
         // setup the output format
-        IngestJob.configureMultiRFileOutputFormatter(configuration, null, null, 0, 0, false);
-
+        IngestJob.configureMultiRFileOutputFormatter(configuration, jobConfig.compression, null, 0, 0, false);
+        log.info("compression type: " + configuration.get("MultiRFileOutputFormatter.compression", "unknown"));
         // all changes to configuration must be before this line
         Job j = Job.getInstance(getConf());
 
         // check if using some form of accumulo in input
         if (jobConfig.inputFiles == null) {
-            // build ranges
-            Collection<Range> ranges = buildRanges(jobConfig.startDate, jobConfig.endDate, jobConfig.splitsPerDay);
+            if (jobConfig.startDate == null) {
+                throw new IllegalArgumentException("startDate cannot be null when inputFiles are not specified");
+            }
+
+            if (jobConfig.endDate == null) {
+                throw new IllegalArgumentException("endDate cannot be null when inputFiles are not specified");
+            }
+
+            if (jobConfig.accumuloMetadata) {
+
+                // fetch the file list by scanning the accumulo.metadata table
+                jobConfig.inputFiles = org.apache.hadoop.util.StringUtils.join(",", getSplitsFromMetadata(accumuloProperties, jobConfig.table,
+                                new Range(new Key(jobConfig.startDate), true, new Key(jobConfig.endDate), true)));
+            } else if (!jobConfig.accumuloData) {
+                // build ranges
+                Collection<Range> ranges = null;
+                if (jobConfig.reprocessEvents) {
+                    ranges = buildRanges(jobConfig.startDate, jobConfig.endDate, jobConfig.splitsPerDay, new Text(), new Text());
+                } else {
+                    ranges = buildFiRanges(jobConfig.startDate, jobConfig.endDate, jobConfig.splitsPerDay);
+                }
 
-            Properties accumuloProperties = builder.build();
+                if (ranges.size() == 0) {
+                    throw new IllegalArgumentException("no ranges created from start: " + jobConfig.startDate + " end: " + jobConfig.endDate);
+                }
+
+                for (Range r : ranges) {
+                    log.info("Range: " + r);
```

Comment:
Can we add a little more context to this message? Maybe plug in the table name or highlight that these are going to be used as input splits. Sometimes our jobs scan so many tables its tough to know what this log line is telling us.